tui/amcrest-tui

```diff
@@ -0,0 +1,205 @@
+import sys
+import tempfile
+import time
+
+if sys.version_info <= (3, 0):  # noqa
+    print("This program requires Python 3 or higher")
+    sys.exit(1)
+
+from configparser import ConfigParser, NoOptionError, NoSectionError
+from argparse import ArgumentParser
+from amcrest import AmcrestCamera
+
+
+class AmcrestTuiViewer():
+
+    AMCREST_CONF = "{0}/.config/amcrest.conf".format(os.path.expanduser("~"))
+
+    def __init__(self,
+                 config_file_name,
+                 channel_snapshot,
+                 timer_between_snapshots,
+                 timeout_error_msg,
+                 merge_snapshots,
+                 framebuffer_device):
+        """
+        config_file_name - optional amcrest config file
+        timer_between_snapshots - timer between each snapshot
+        """
+        if not os.path.exists(self.AMCREST_CONF):
+            raise RuntimeError(
+                "Please set the initial {0}".format(self.AMCREST_CONF)
+            )
+
+        # Timeout settings
+        self.timer_between_snapshots = timer_between_snapshots
+        self.timeout_error_msg = timeout_error_msg
+
+        # Channel for snapshot
+        self.channel_snapshot = channel_snapshot
+
+        # FIXME: User might be able to provide config file path
+        self.config_file_name = config_file_name
+
+        # FIXME: Add such feature via convert tool
+        self.merge_snapshots = merge_snapshots
+        self.fb_device = framebuffer_device
+
+        # Setting config stuff for amcrest.conf
+        self.config = ConfigParser()
+        self.config.read(self.AMCREST_CONF)
+
+        # Creating dict to hold cameras data from config file
+        self.amcrest_config = {}
+
+        # Filling the dictionary amcrest_config with all cameras available
+        for camera_number in range(len(self.config.sections())):
+
+            camera_name = self.config.sections()[camera_number]
+
+            try:
+                self.amcrest_config[camera_name] = {
+                    'hostname': self.config.get(camera_name, 'hostname'),
+                    'port':     self.config.get(camera_name, 'port'),
+                    'username': self.config.get(camera_name, 'username'),
+                    'password': self.config.get(camera_name, 'password'),
+                    'device':   AmcrestCamera(
+                        host=self.config.get(camera_name, 'hostname'),
+                        port=self.config.get(camera_name, 'port'),
+                        user=self.config.get(camera_name, 'username'),
+                        password=self.config.get(
+                            camera_name, 'password')).camera
+                }
+            except (NoSectionError, NoOptionError) as e:
+                print(("ERROR! %s found at %s" % (e, self.AMCREST_CONF)))
+                raise
+
+    def _view_image(self, image):
+        try:
+            cmd = subprocess.Popen(
+                ['fbi', '-a', '-noverbose', '-d', self.fb_device, image],
```

Comment:
What about to create a try/exception if the user does not have the `fbi` command? 

```bash
# ./amcrest-tui
Loading new image from: amcrestcam1
Traceback (most recent call last):
  File "./amcrest-tui", line 205, in <module>
    tui_viewer.run()
  File "./amcrest-tui", line 133, in run
    self._view_image(tmpfile.name)
  File "./amcrest-tui", line 99, in _view_image
    shell=False
  File "/usr/lib64/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/usr/lib64/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'fbi'
```

To address, I've installed fbi on my Fedora

```
↪ rpm -aq | grep fbida
fbida-2.09-10.fc24.x86_64
```